packages/components/document-card/stories/document-card.stories.js

```diff
@@ -0,0 +1,16 @@
+import { storiesOf, html } from '@open-wc/demoing-storybook';
+import { withKnobs, text } from '@storybook/addon-knobs';
+
+import '@tradeshift/elements.document-card';
+
+storiesOf('ts-document-card', module)
+	.addDecorator(withKnobs)
+	.add('default', () => {
+		const name = text('name', 'Invoice #321');
+		const description = text('description', 'CompanyName');
+		const mobileDescription = text('mobileDescription', 'Description');
+		return html`
+			<ts-document-card ?name="${name}" ?description="${description}" ?mobileDescription="${mobileDescription}">
```

Comment:
`?` should be used only for boolean attributes, so you can remove them here.

Reply:
ok